### PR TITLE
Login ig saving data in db + fix bugs

### PR DESCRIPTION
**Pull request by ferdefiore**
+Use the db to save IG login information
+Bugs fix:
-Blocking writing in birthday field
-The app was crashing when the email and birthday want to update in the db and at the same time, because was async, another method was triying to send an email to the same user that has been updating.
-Delete a tost from the method SendMail that crash the app because it was triying to show a toast in a view that it has been destroyed.

### CODE REVIEW

File: app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java [hunk 1]
```diff
@@ -47,7 +49,10 @@ public void onPostForHandleUrl(MyWVClient.PostForHandleUrl event) {
     @Subscribe
     public void onRetIgInformation(InstagramLoginModel.RetIgInformation event) {
         view.closeDialog();
+        String names[] = event.name.split(" ",2);
```
Comment (SartiMau) [inline app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java]: Put this in a constant please

File: app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java [hunk 2]
```diff
@@ -47,7 +49,10 @@ public void onPostForHandleUrl(MyWVClient.PostForHandleUrl event) {
...
+        String id = String.format("IG%s", event.id);
```
Comment (SartiMau) [inline app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java]: same

File: app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java [hunk 3]
```diff
@@ -47,7 +49,10 @@ public void onPostForHandleUrl(MyWVClient.PostForHandleUrl event) {
...
         bus.post(new InformationReady(event.name, event.logeado, event.id));
+        bus.post(new IgUserDataRecoveredEvent(User.getUser(names[0], names[1], null, null, id)));
```
Comment (SartiMau) [inline app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java]: same

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java (hunk 2 above)
Comment (ferdefiore) [reply app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java]: done

File: app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java (hunk 1 above)
Comment (ferdefiore) [reply app/src/main/java/com/pps/globant/fittracker/mvp/presenter/InstagramLoginPresenter.java]: done

**Review by SartiMau — APPROVED** [review]

**Review by capraber — APPROVED** [review]

**Pull request merged**